requirements/base.txt

```diff
@@ -3,9 +3,8 @@ torch>=1.8.*, <=1.11.0
 tqdm>=4.57.0, <=4.63.0
 PyYAML>=5.4, <=6.0
 fsspec[http]>=2021.05.0, !=2021.06.0, <=2022.2.0
-tensorboard>=2.2.0, <2.10.0
+tensorboard>=2.9.1, <2.10.0
```

Comment:
Only for `oldest` tests to pass in our CI, we need to update the lower bound of `tensorboard`.